src/components/SidebarMobile/SidebarMobile.jsx

```diff
@@ -70,7 +76,7 @@ export default class SidebarMobile extends React.Component {
             key={ absoluteUrl }
             to={ absoluteUrl }
             onClick={ this._close.bind(this) }>
-            <h3>{ section.title }</h3>
+            <h3>{ section.title || section.url }</h3>
```

Comment:
Why fall back on `section.url`? Shouldn't all `section.title`s be populated?

Reply:
for something don't have a title as: https://github.com/webpack/webpack.js.org/blob/feat-webpack2/antwar.config.js#L150